contracts-core/periphery-typechain/UmbraBatchSend.ts

```diff
@@ -0,0 +1,228 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
We should avoid committing these autogenerating files, and they should be generated by the build command

Reply:
Removed for now!